molecule/kind/molecule.yml

```diff
@@ -34,7 +34,7 @@ provisioner:
         migration_namespace: ${TEST_MIGRATION_NAMESPACE:-openshift-migration}
         namespace: ${TEST_OPERATOR_NAMESPACE:-openshift-migration}
         kubeconfig_file: ${MOLECULE_EPHEMERAL_DIRECTORY}/kubeconfig
-        test_image: ${REPLACE_IMAGE:-quay.io/konveyor/mig-operator-container:release-1.5.1}
+        test_image: ${REPLACE_IMAGE:-quay.io/konveyor/mig-operator-container:release-1.5.2}
```

Comment:
I think we need to update this in the legacy operator as well. It might be `latest` on the 1.5.2 branch, but should get changed.